ci/code_checks.sh

```diff
@@ -65,1541 +65,1553 @@ fi
-        pandas.tseries.offsets.QuarterBegin.copy\
-        pandas.tseries.offsets.QuarterBegin.freqstr\
-        pandas.tseries.offsets.QuarterBegin.kwds\
-        pandas.tseries.offsets.QuarterBegin.name\
-        pandas.tseries.offsets.QuarterEnd.copy\
-        pandas.tseries.offsets.QuarterEnd.freqstr\
-        pandas.tseries.offsets.QuarterEnd.kwds\
-        pandas.tseries.offsets.QuarterEnd.name\
-        pandas.tseries.offsets.Second.copy\
-        pandas.tseries.offsets.Second.freqstr\
-        pandas.tseries.offsets.Second.kwds\
-        pandas.tseries.offsets.Second.name\
-        pandas.tseries.offsets.Second.nanos\
-        pandas.tseries.offsets.SemiMonthBegin\
-        pandas.tseries.offsets.SemiMonthBegin.copy\
-        pandas.tseries.offsets.SemiMonthBegin.freqstr\
-        pandas.tseries.offsets.SemiMonthBegin.kwds\
-        pandas.tseries.offsets.SemiMonthBegin.name\
-        pandas.tseries.offsets.SemiMonthEnd\
-        pandas.tseries.offsets.SemiMonthEnd.copy\
-        pandas.tseries.offsets.SemiMonthEnd.freqstr\
-        pandas.tseries.offsets.SemiMonthEnd.kwds\
-        pandas.tseries.offsets.SemiMonthEnd.name\
-        pandas.tseries.offsets.Tick.copy\
-        pandas.tseries.offsets.Tick.freqstr\
-        pandas.tseries.offsets.Tick.kwds\
-        pandas.tseries.offsets.Tick.name\
-        pandas.tseries.offsets.Tick.nanos\
-        pandas.tseries.offsets.Week.copy\
-        pandas.tseries.offsets.Week.freqstr\
-        pandas.tseries.offsets.Week.kwds\
-        pandas.tseries.offsets.Week.name\
-        pandas.tseries.offsets.WeekOfMonth\
-        pandas.tseries.offsets.WeekOfMonth.copy\
-        pandas.tseries.offsets.WeekOfMonth.freqstr\
-        pandas.tseries.offsets.WeekOfMonth.kwds\
-        pandas.tseries.offsets.WeekOfMonth.name\
-        pandas.tseries.offsets.YearBegin.copy\
-        pandas.tseries.offsets.YearBegin.freqstr\
-        pandas.tseries.offsets.YearBegin.kwds\
-        pandas.tseries.offsets.YearBegin.name\
-        pandas.tseries.offsets.YearEnd.copy\
-        pandas.tseries.offsets.YearEnd.freqstr\
-        pandas.tseries.offsets.YearEnd.kwds\
-        pandas.tseries.offsets.YearEnd.name\
-        pandas.util.hash_array\
-        pandas.util.hash_pandas_object # There should be no backslash in the final line, please keep this comment in the last ignored function
-    RET=$(($RET + $?)) ; echo $MSG "DONE"
+    # Run multiple calls of the validation script in a single python python process
+    # Each group of parameters is separated with --
+    # This way module compiles, imports and processed source files can be cached, among other things
+    PARAMETERS=(--format=actions --errors=EX01,EX03,EX04,GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,PD01,PR03,PR04,PR05,PR06,PR08,PR09,PR10,RT01,RT02,RT04,RT05,SA02,SA03,SA04,SA05,SS01,SS02,SS03,SS04,SS05,SS06)
+    #####
+    # end of full error validation group
+    #####
+    PARAMETERS+=(--for_error_ignore_functions PR02 \
```

Comment:
I'd personally use:
```
$BASE_DIR/scripts/validate_docstrings.py --format=actions --ignore \
    pandas.Series.dt.as_unit,PR01,PR02,GL08 \
    pandas.Series.dt.to_period,PR02,RT07 \
    ...
```

I think it's clearer, more compact, more difficult to make mistakes and easier to review changes when something is fixed. What do you think?